### PR TITLE
[reproc]: Update to v11.0.1.

### PR DESCRIPTION
**Pull request by daandemeyer**

### CODE REVIEW

**Comment by PhoebeHui** [comment]
@DaanDeMeyer, thanks for the PR!

It failed on x86-windows on CI testing, could you help have a look?

**Comment by DaanDeMeyer** [comment]
I fixed x86, the other failures seem unrelated.

**Review by PhoebeHui — APPROVED** [review]

File: ports/reproc/portfile.cmake [hunk 1]
```diff
@@ -24,10 +24,10 @@ foreach(TARGET reproc reprocxx)
         CONFIG_PATH share/${TARGET} 
         TARGET_PATH share/${TARGET}
     )
+    
+    file(
+        INSTALL ${SOURCE_PATH}/LICENSE 
+        DESTINATION ${CURRENT_PACKAGES_DIR}/share/${TARGET}
+        RENAME copyright
+    )
```
Comment (ras0219-msft) [inline ports/reproc/portfile.cmake]: Moving the `file(INSTALL ...)` into the `foreach` loop seems completely unneeded; was this intentional?

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

File: ports/reproc/portfile.cmake (hunk 1 above)
Comment (DaanDeMeyer) [reply ports/reproc/portfile.cmake]: It seemed like it would make stuff more consistent between reproc and reproc++. I can remove it again if needed.

**Comment by ras0219-msft** [comment]
It looks like this upgrade got picked up by our mass-upgrade script:
https://github.com/microsoft/vcpkg/commit/08c951fef9de63cde1c6b94245a63db826be2e32

Please open a new PR if there are any remaining issues!

**Pull request closed without merging**